docs/usage/configuration-templates.md

```diff
@@ -72,3 +72,23 @@ You can change the PR body in the following ways:
 - Add a footer by using `prFooter`
 - Add a note by using `prBodyNotes`
 - Edit the embedded table by using `prBodyDefinitions` and `prBodyColumns`
+
+### Renovate tags
+
+This tags are only available by a subset of the platforms where html tags are supported, e.g., GitHub, GitLab, ...
+
+#### renovate:start and renovate:end
+
+The Renovate specific description in a new pull requests will be wrapped by the tags e.g.,
+
+```md
+<!--renovate:start-->
+
+Renovate specific body
+
+<!--renovate:end-->
+```
+
+This allows users to extend the description outside of this tags with custom information without losing the changes in the pull request description upon an automated pull request update by the Renovate Bot. Only the part between the two tags will be replaced with an update Renovate body.
+
+In case only one of the two tags is available, either all text after `renovatge:start` or before `renovate:end` is replaced with the Renovate specific body.
```

Comment:
```suggestion
If only one of the two tags is available then either all text after `renovate:start` or before `renovate:end` is replaced with the Renovate specific body.
```

Reply:
Thank you for the suggestions. I have now simplified the wording to the bare minimum.
54ca21b3f314c74270ccd28424400b8d2829c819